app/domain/streams/message_processor_job.rb

```diff
@@ -14,9 +14,20 @@ def perform(payload, routing_key)
 
       Monitor::Messages.run(routing_key)
 
-      ActiveRecord::Base.transaction do
-        message.handler.process
+      begin
+        ActiveRecord::Base.transaction do
+          message.handler.process
+        end
+      rescue StandardError => e
+        GovukError.notify(e)
+        logger.error(e.message)
```

Comment:
How does this compare to changing the logger that Sidekiq uses? Either just changing the log level, or changing the formatter?

Reply:
We are trying to get rid of the stack trace

Reply:
I get that, I'm asking about the approach.

One difference I think, is that this approach might not work with Sidekiq retrying? If the exception is being caught, does Sidekiq know the job has failed?